install/kubernetes/helm/istio/charts/grafana/dashboards/pilot-dashboard.json

```diff
@@ -743,7 +743,7 @@
           "refId": "G"
         },
         {
-          "expr": "sum(rate(pilot_xds_push_errors{job=\"pilot\"}[1m]))",
+          "expr": "sum(rate(pilot_xds_push_errors{job=\"pilot\"}[1m])) by (type)",
```

Comment:
iirc, the pilot code updates `pilot_xds_pushes` with `type`s that contain `_senderr` and `_builderr` (for all but `eds`). This change will hide all `_senderr` and `_builderr`, as they aren't in the `pilot_xds_push_errors` (unless something has changed very recently).

Reply:
I thought we always incremented `pilot_total_xds_internal_errors`, but after closer inspection this is not the case. In some cases we do, in some cases we don't. In some cases we do and then `panic` immediately after, pretty much guaranteeing we never see the metrics anyways. Right now we have builderrs shown but not senderrs. 

I would much rather have all the errors in the error panel. Unless your config is just horribly broken somehow, the successful pushes will dominate the errors and they will be hidden.

I think there is some duplication on the dashboard as well. I may take a stab at making some more changes as well to clean it up + the metrics